ow_regolith/include/ServiceClientFacade.h

```diff
@@ -0,0 +1,80 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#ifndef SERVICE_CLIENT_FACADE_H
+#define SERVICE_CLIENT_FACADE_H
+
+#include <memory>
+
+#include <ros/ros.h>
+
+namespace ow_regolith {
+
+template <class T>
+class ServiceClientFacade
+{
+public:
+  ServiceClientFacade() : m_node_handle(nullptr)
+  {
+    // do nothing
+  }
+
```

Comment:
Good to 'default' or 'delete' the copy constructor, assignment operator, and destructor, as applicable, if you're not defining them.

Reply:
Thank you for the consistent class method hygiene reminders! I'll catch onto this one eventually. :laughing: